index.js

```diff
@@ -32,8 +30,22 @@ function dispatcher (handlers) {
   start.model = setModel
   start.state = getState
   start.start = start
+  start.use = useHooks
   return start
 
+  // push an object of hooks onto an array
+  // obj -> null
+  function useHooks (hooks) {
+    assert.equal(typeof hooks, 'object', 'barracks.use: hooks should be an object')
```

Comment:
Haha, you're obv right... Except that it's valid to append properties onto an array and so it would parse as expected 😂. Not that we should encourage that but, JS is def a weird language. #keepjavascriptweird